src/zc/c_lbu.adoc

```diff
@@ -0,0 +1,48 @@
+<<<
+[#insns-c_lbu,reftext="Load unsigned byte, 16-bit encoding"]
+==== c.lbu
+
+Synopsis::
+Load unsigned byte, 16-bit encoding
+
+Mnemonic::
+c.lbu _rd'_, _uimm_(_rs1'_)
+
+Encoding (RV32, RV64)::
+[wavedrom, , svg]
+....
+{reg:[
+    { bits:  2, name: 0x0, attr: ['C0'] },
+    { bits:  3, name: 'rd\'' },
+    { bits:  2, name: 'uimm[0|1]' },
+    { bits:  3, name: 'rs1\'' },
+    { bits:  3, name: 0x0 },
+    { bits:  3, name: 0x4, attr: ['FUNCT3'] },
+],config:{bits:16}}
+....
+
+include::c_lbsb_imm_offset.adoc[]
+
+Description::
+This instruction loads a byte from the memory address formed by adding _rs1'_ to the zero extended immediate _uimm_. The resulting byte is zero extended to XLEN bits and is written to _rd'_. 
+
+[NOTE]
+====
+  _rd'_ and _rs1'_ are from the standard 8-register set x8-x15.
+====
+
+Prerequisites::
+None
+
+32-bit equivalent::
+<<insns-lbu>>
```

Comment:
broken xref

Reply:
How do you want to handle these?  I believe they were placeholders.  There is quite a few.

Reply:
is the intention to eventually have separate instruction pages for each instruction?
if so then eventually the xref will point somewhere. If not then just the name of teh 32-bit instruction would do it

Reply:
I have no idea what the intention is for separate instructions, however, leaving xrefs as they are will require whoever authors the 32-bit instructions to make sure they add the same anchor to make these xrefs work.  I think for now I should remove each 32-bit instruction section.  @tariqkurd-repo would you agree with that?